### PR TITLE
Code coverage improvement part1

### PR DESCRIPTION
**Pull request by DragosDanielBoia**
1.Code coverage improvement part1

### CODE REVIEW

File: src/PullRequestQuantifier.Repository.Service/appsettings.Development.json [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
...f1r1iKP3HMOhx/v9UN67hPafzYdTQ0uP308vQDWKX03G5oC3C2bpxQ4JIbt4no5mzPgnXIxMtIS8EWa9h2q0toTg5TuICBD29xz8aPYq5GMq5WxBNqPFMomZyhr+/2jvYAK5T4KhnfPiGCQQdpgbuBsUH3D8w/czuHX+A10k3lCQwr2I2b7eS0qJgandUQg2Qp/KwKvE5VTl1J0G+WDF9kkys4p+p2yGfwZGFUu2CHdGA+tG8PC42FVIUwREnm/rZi9leAW9atUrFBHCx/1qTooCZ+UIupXCTSS4qYKg2UT4AYhLeV3b1utXLpWcCpAoGBAPguHx4hopfOzurqh6QtvsQtmtsWdaXMQTSDTOud01pQPuN/gdgCVM1507HzNvSuX38dx9xkzpb7j8rekqYuIgpxiUTu4i102rbU724vuqDoDbUdTyalq8OB7Na+tIC2WHtzRbsCi2I2aB1sfvlh1d3dfN36aiSqAfbJt2juh1gLAoGBAOX45CR8yWFrR/+ubhNLarAIp19kRS0o7tzlLg4IWzQnkSNy1qUJfVY+GL9QGMyP/2uAINDNsDQ8iKELaOjAiNpuX7uV/ll7p/rz0mp7/ZEfm8UMdAnJPmRgK8/HcJpoHfUPJwimuKPfpI5s8jNoeZwhTwpI7ceKS1mLJPNi5ZYbAoGBAJko9YFF5q7jLWVimZUjuhAdyOyeXrqUZEVBJq5d9zpMPZNudDZXsfgg0BmI7IhyOHMdcOM9vxUEaMgdNcmivNVPHhTRi/nbnMKqhJyujfGRZCw6y0chhPAh9YTNql2oqtB84jK4LkbNJtHJzE6hJzhF4TKowOqhlJa2BB+Y3SPjAoGAPYGrwvZ9BckjoIaOeiqZhitKWIEA32srN3k0MBycB/7tGoYEUm+PThmdlM1fVJbOwXmrTS02C0KFaE0c6eNUttb+U7VyKYyhCMVdaY1v6/iAONtzzYo9Xas7DH9S5nkiSgCbd/zfNTUDVfRdctJmdBOUiDOpJM0gCfrlDxNMRDECgYA03LeodkEpdpecl3ZcuP+78Oo8N+J7vlKBZQmIhCX36c7NFWy4ElmKU9eUDRqVSRZuZcxHflPaDtr2s3L9hh1TGyv3zlnBh3N96Mz+ZPUNWBTkS5Q4kugYrWfkCvHcIQjMhXOxEr6BBX0w7+3a5iT+SYhjNLrknMXh6FkFUgJEqg==",
```
Comment (goelhardik) [inline src/PullRequestQuantifier.Repository.Service/appsettings.Development.json:24]: Remove the keys? I think we'll have to re-generate these if they can't be purged from branch history.

**Review by goelhardik — APPROVED** [review]

File: src/PullRequestQuantifier.Repository.Service/appsettings.Development.json (hunk 1 above)
Comment (goelhardik) [reply src/PullRequestQuantifier.Repository.Service/appsettings.Development.json:24]: Oh didn't realize auto-merge was enabled.

Comment (goelhardik) [reply src/PullRequestQuantifier.Repository.Service/appsettings.Development.json:24]: Revoking the secrets.

Comment (DragosDanielBoia) [reply src/PullRequestQuantifier.Repository.Service/appsettings.Development.json:24]: uau, ok let me remove it, one sec, totally didn't realize

Comment (goelhardik) [reply src/PullRequestQuantifier.Repository.Service/appsettings.Development.json:24]: Revoked everything. Appconfig is setup for repository service dev. Let's use that then we don't need to put settings locally.

**Pull request merged**